mlrun/utils/notifications/notification/webhook.py

```diff
@@ -63,6 +63,33 @@ async def push(
             request_body["custom_html"] = custom_html
 
         if override_body:
+            list_edit_runs = []
+            for run in runs:
+                if isinstance(run, mlrun.model.RunObject):
```

Comment:
Can also be RunTemplate
```suggestion
                if hasattr(run,"to_dict):
```